code/modules/mob/living/simple_animal/hostile/terror_spiders/terror_spiders.dm

```diff
@@ -0,0 +1,328 @@
+			spider_specialattack(G,can_poison)
+		else
+			G.attack_animal(src)
+	else
+		target.attack_animal(src)
+
+/mob/living/simple_animal/hostile/poison/terror_spider/proc/spider_specialattack(var/mob/living/carbon/human/L, var/poisonable)
+	L.attack_animal(src)
+
+
+
+
+
+
+// --------------------------------------------------------------------------------
+// --------------------- TERROR SPIDERS: PROC OVERRIDES ---------------------------
+// --------------------------------------------------------------------------------
+
+
+/mob/living/simple_animal/hostile/poison/terror_spider/examine(mob/user)
+	..()
+	var/msg = ""
+	if(stat == DEAD)
+		msg += "<span class='deadsay'>It appears to be dead.</span>\n"
+	else
+		if(key)
+			msg += "<BR><span class='warning'>Its eyes regard you with a curious intelligence.</span>"
+		if(health > (maxHealth*0.95))
+			msg += "<BR><span class='notice'>It is in excellent health.</span>"
+		else if(health > (maxHealth*0.75))
+			msg += "<BR><span class='notice'>It has a few injuries.</span>"
+		else if(health > (maxHealth*0.55))
+			msg += "<BR><span class='warning'>It has many injuries.</span>"
+		else if(health > (maxHealth*0.25))
+			msg += "<BR><span class='warning'>It is barely clinging on to life!</span>"
+		else if(health < maxHealth && regen_points > regen_points_per_kill)
+			msg += "<BR><span class='notice'>It appears to be regenerating quickly</span>"
+		if(killcount == 1)
+			msg += "<BR><span class='warning'>It is soaked in the blood of its prey.</span>"
+		if(killcount > 1)
+			msg += "<BR><span class='warning'>It is soaked with the blood of " + num2text(killcount) + " prey it has killed.</span>"
+	to_chat(usr,msg)
+
+
+/mob/living/simple_animal/hostile/poison/terror_spider/New()
+	..()
+	ts_spiderlist += src
+	if(type == /mob/living/simple_animal/hostile/poison/terror_spider)
+		message_admins("[src] spawned in [get_area(src)] - a subtype should have been spawned instead.")
+		qdel(src)
+	else
+		add_language("TerrorSpider")
+		add_language("Galactic Common")
+		default_language = all_languages["TerrorSpider"]
+
+		name += " ([rand(1, 1000)])"
+		msg_terrorspiders("[src] has grown in [get_area(src)].")
+		if(name_usealtnames)
+			name = pick(altnames)
+		if(is_away_level(z))
+			spider_awaymission = 1
+			ts_count_alive_awaymission++
+		else
+			ts_count_alive_station++
+		// after 30 seconds, assuming nobody took control of it yet, offer it to ghosts.
+		spawn(150) // deciseconds!
+			CheckFaction()
+		spawn(300) // deciseconds!
+			if(spider_awaymission)
+				return
+			else if(ckey)
+				notify_ghosts("[src] has appeared in [get_area(src)]. (already player-controlled)")
+			else if(ai_playercontrol_allowingeneral && ai_playercontrol_allowtype)
+				notify_ghosts("[src] has appeared in [get_area(src)].", enter_link = "<a href=?src=\ref[src];activate=1>(Click to control)</a>", source = src)
+
+/mob/living/simple_animal/hostile/poison/terror_spider/Destroy()
+	ts_spiderlist -= src
+	..()
```

Comment:
`return ..()` is the correct format. Your does the same thing, but it's inconsistent with the majority of `Destroy`s.